gno.land/pkg/integration/doc.go

```diff
@@ -21,6 +21,21 @@
 //   - Creates a new user in the default keybase directory.
 //   - Must be run before `gnoland start`.
 //
+// 4. `use`:
+//   - Loads a specific package from the example folder or from the working ($WORK) directory.
+//   - Can be used to load a single package or all packages within a directory.
+//   - The command takes either one or two arguments. The first argument is the name of the package(s),
+//     and the second (optional) argument is the path to the package(s).
+//     Examples:
+//     -- # Load a package from the example packages directory:
+//     -- use gno.land/p/demo/ufmt
+//     -- # Load a package `./bar` from the current testscript's working directory with the name `gno.land/r/foobar/bar`:
+//     -- use gno.land/r/foobar/bar $WORK/bar
+//   - If the path is not prefixed with the working directory, it is assumed to be relative to the examples directory.
+//   - It's important to note that the load order is significant when using
+//     multiple use command; packages should be loaded in the order they are
+//     dependent upon.
```

Comment:
note: maybe we should specify that `use` commands can only happen before `gnoland start`, if I understood its functioning correctly?

Reply:
Rename `use` to `copy` (to match Dockerfile) or `add-to-genesis` or choose another word that better conveys that it's primarily a filesystem manipulation rather than a command.